ui/app/AppLayouts/Chat/popups/community/CreateChannelPopup.qml

```diff
@@ -41,8 +41,10 @@ StatusModal {
     onClosed: destroy()
 
     function isFormValid() {
-        return contentItem.channelName.valid &&
-               contentItem.channelDescription.valid
+        return (scrollView.channelName.valid &&
+               scrollView.channelDescription.valid &&
+               scrollView.channelName.text.length > 0 &&
+               scrollView.channelDescription.text.length > 0)
```

Comment:
These `StatusInput`s both have a `StatusMinlengthValidator` with a min length of `1` attached to it, so these additional checks should no be needed.

Can you verify this?

Reply:
yes I know, it doesn't work when opening the popup as `valid` is always `true`. As soon as text is typed then validators work as expected.

Reply:
We have a validation mode property to run validation even when the field is not dirty

https://github.com/status-im/StatusQ/blob/master/src/StatusQ/Controls/StatusInput.qml#L43

Reply:
@PascalPrecht thanks for pointing this out! updated the commit and also rebased :)

Reply:
Awesome!